parsers/AR.py

```diff
@@ -20,14 +20,32 @@
 )
 CAMMESA_RENEWABLES_ENDPOINT = "https://cdsrenovables.cammesa.com/exhisto/RenovablesService/GetChartTotalTRDataSource/"
 
+CAMMESA_RENEWABLES_REGIONAL_ENDPOINT = (
+    "https://cdsrenovables.cammesa.com/exhisto/RenovablesService/getCapacidadesRegiones"
+)
+
+CAMMESA_REGIONS_ENDPOINT = "https://api.cammesa.com/demanda-svc/demanda/RegionesDemanda"
```

Comment:
nit: Maybe format the constants the same way?

Reply:
The parentheses were added/removed automatically by the linter! Unless you mean to format the name of the constants the same?